src/components/Navbar/links.tsx

```diff
@@ -10,6 +10,7 @@ export const navLinks: NavLink[] = [
   { title: "Events", slug: "events" },
   { title: "Visit Us", slug: "visit" },
   { title: "Contact Us", slug: "contact" },
+  { title: "Profile", slug: "profile"},
   // { title: "Aksara", slug: "aksara", alt: true },
```

Comment:
Also what does `alt` mean in the Aksara portion here

Reply:
Alt controls the color of the other unchosen sections. E.g. When I'm in Profile, the color of profile is red and the color of the rest is controlled by alt. If alt is true, then the color will be black, while if alt is false, the color will be white.

Reply:
yup nice one!